client/logic/ui/map_editor_ui.gd

```diff
@@ -260,7 +264,7 @@ func _unhandled_input(event: InputEvent) -> void:
 			_update_money_container()
 
 func _on_money_spin_value_changed(value: float) -> void:
-	map.get_player(player_option_button.selected).money = value
+	map.get_player(player_option_button.selected).money = round(value)
```

Comment:
there is no better way to avoid this round(value)? We get lots of them in this P/R

Reply:
What would you suggest?